src/reanimated2/animations.js

```diff
@@ -174,7 +174,7 @@ export function withSpring(toValue, userConfig, callback) {
 
       const isOvershooting = () => {
         if (config.overshootClamping && config.stiffness !== 0) {
-          return current < toValue ? current > toValue : current < toValue;
```

Comment:
I read this to return false all the time, am I correct?

Reply:
Yup, A < B and A >(=) B are mutually exclusive, so if `current < toValue` is false you return same expression (so false) and if it's true you return inverted expression (which is false in that case)